index.html

```diff
@@ -45,11 +45,11 @@ <h1 class="name">I'm Ifeanyi oti, Glad to see you!</h1>
                   <p class="lets-connect1">LET’S CONNECT</p>                
                 <div class="social-links-container">
                 <ul class="social-media1">
-                    <li><a href="#"><img src="images/twitter-icon.svg" alt=""></a></li>
-                    <li><a href="#"><img src="images/Linkedin icon.svg" alt=""></a></li>
-                    <li><a href="#"><img src="images/M-logo.svg" alt=""></a></li>
-                    <li><a href="#"><img src="images/github-icon.svg" alt=""></a></li>
-                    <li><a href="#"><img src="images/index-finger.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/twitter-icon.svg" alt="Twitter icon"></a></li>
+                    <li><a href="#"><img src="images/Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                    <li><a href="#"><img src="images/M-logo.svg" alt="Medium icon"></a></li>
+                    <li><a href="#"><img src="images/github-icon.svg" alt="Github icon"></a></li>
+                    <li><a href="#"><img src="images/index-finger.svg" alt="Angellist icon"></a></li>
```

Comment:
- [ ] It will be better if your alt text doesn't contain `icon` text.
> Good example `alt="Twitter"`. Kindly change the rest of the alt text because Screen readers automatically provide that information.

Reply:
Hi, @omarsalem7  thanks for your feedback, 
I'll add the alt text to the remaining images